code/surface_area/compute_surface_area.py

```diff
@@ -0,0 +1,172 @@
+import argparse
+from pathlib import Path
+from typing import Iterable
+
+from niwrap import workbench as wb
+from styxdocker import DockerRunner
+
+from validate_surface_files import validate_output_file_data
+from utils import find_surface_files, get_map_info
+
+my_runner = DockerRunner()
+my_runner.data_dir = Path(__file__).parent / "../../share/Inputs"
```

Comment:
Again, not a big deal here, but would move normally move this into `main` and have it set up there. That way you can also set the `runner.data_dir` from the input arguments.